doc/source/examples/unresolved-cfd-dem/boycott-effect/boycott-effect.rst

```diff
@@ -0,0 +1,372 @@
+==================================
+Boycott Effect
+==================================
+
+It is strongly recommended to visit `DEM parameters <../../../parameters/dem/dem.html>`_  and `CFD-DEM parameters <../../../parameters/unresolved-cfd-dem/unresolved-cfd-dem.html>`_ for more detailed information on the concepts and physical meaning of the parameters ind DEM and CFD-DEM.
+
+Features
+----------------------------------
+- Solvers: ``dem_3d`` and ``cfd_dem_coupling_3d``
+- Three-dimensional problem
+- Displays the selection of models and physical properties.
+- Simulates a solid-liquid sedimentation.
+
+
+Files used in this example
+---------------------------
+``/examples/unresolved-cfd-dem/boycott-effect``
+
+
+Description of the case
+-----------------------
+
+This example simulates the sedimentation of a group of particles in a viscous fluid. Two cases where simulated. In the first case, the tube was vertically placed. In the second case, the tube was inclined at a :math:`20^{\circ}` with the vertical. First, we use Lethe-DEM to insert the particles. We enable check-pointing in order to write the DEM checkpoint files which will be used as the starting point of the CFD-DEM simulation. Then, we use the ``cfd_dem_coupling_3d`` solver within Lethe to simulate the sedimentation of particles by initially reading the checkpoint files from the DEM simulation.
+
+
+DEM parameter file
+-------------------
+
+The syntax is flexible. Parameters do not need to be specified in a specific order, but only within the subsection in which they belong. All parameter subsections are described in the `parameter section <../../../parameters/parameters.html>`_ of the documentation.
+
+To set-up the sedimentation case, we first insert the particles. 
+
+We introduce the different sections of the parameter file ``dem-packing-in-fluidized-bed.prm`` needed to run this simulation. Most subsections are explained in detail in other CFD-DEM examples such as:  `Gas-solid spouted bed <../../../examples/unresolved-cfd-dem/gas-solid-spouted-bed/gas-solid-spouted-bed.html>`_ Therefore, we will not go over them in detail.
+
+Mesh
+~~~~~
+
+In this example, we are simulating a rectangular tube. We use the deal.II GridGenerator in order to generate a hyper rectangle that is subdivided along its height. The following portion of the DEM parameter file shows the function called:
+
+.. code-block:: text
+
+    subsection mesh
+ 	set type                 				= dealii
+    	set grid type            				= subdivided_hyper_rectangle
+    	set grid arguments       				= 15,70,15:-0.03,0,-0.03:0.03,0.4,0.03:true
```

Comment:
Is there reasons for this geometry? Is it arbitrary or from an online example?

Reply:
Just arbitrary. Lucka is doing something similar for resolved cfd-dem so I followed the same geometry structure